ROADMAP.md

```diff
@@ -8,59 +8,65 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 We use the [milestone](https://github.com/rook/rook/milestones) feature in Github so look there for the most up-to-date and issue plan.
 
-## Rook 1.2
 
-* Integration Tests
-  * Speed up integration tests [#1218](https://github.com/rook/rook/issues/1218)
+## Rook 1.4
+
+The following features are targeted for Rook v1.4 (July 2020).
+
 * Ceph
+  * Admission controller [#4819](https://github.com/rook/rook/issues/4819)
+  * RGW Multi-site replication (experimental) [#1584](https://github.com/rook/rook/issues/1584)
   * Handle IPv4/IPv6 dual stack configurations [#3850](https://github.com/rook/rook/issues/3850)
-  * Simplify metadata backup and disaster recovery [#592](https://github.com/rook/rook/issues/592)
-  * Add support for priority classes [#2787](https://github.com/rook/rook/issues/2787)
-  * Allow fine-grained management of cephx users with a Client CRD [#3175](https://github.com/rook/rook/issues/3175)
-  * Gather Ceph crash reports automatically, to be viewed in the Ceph dashboard [#2882](https://github.com/rook/rook/issues/2882)
-  * Expose a Ceph Client CRD to create capabilities for external usage [#3175](https://github.com/rook/rook/issues/3175)
-  * Update the Ceph-CSI driver to the latest version (tentatively v2.0)
-* EdgeFS
-  * Improvements for single node clusters
-  * Add support for payloads on external S3 server [#4431](https://github.com/rook/rook/issues/4431)
-  * Add support for rtkvs disk engine and Samsung KVSSD [#3997](https://github.com/rook/rook/issues/3997)
-* YugabyteDB
+  * Multus networking configuration declared stable
+  * RBD Mirroring configured with a CRD
+  * All Ceph controllers updated to the controller runtime
+  * Uninstall options for sanitizing OSD devices
+
+## Themes
 
-## Future improvements
+The general areas for improvements include the following, though may not be committed to a release.
 
-* Formalize the Rook API
-* Custom resource validation, progress, status [#1539](https://github.com/rook/rook/issues/1539)
-* Integration testing improvements
-  * Update promotion and release channels to align with storage provider specific statuses [#1885](https://github.com/rook/rook/issues/1885)
-  * Include more environments in test pipeline [#1841](https://github.com/rook/rook/issues/1841)
-* Support for dynamic provisioning of database [#1704](https://github.com/rook/rook/issues/1704) storage types
-* Update Rook controllers to build on the controller runtime [#1981](https://github.com/rook/rook/issues/1981)
-* Wildcard support for disk selection spec [#1744](https://github.com/rook/rook/issues/1744)
+* Admission Controllers
```

Comment:
For other operators?

Reply:
Yes, work can be done for other operators to build their admission controllers. This isn't committing the feature for other operators, just that it's an expected improvement as the operators mature.